docs/source/en/_toctree.yml

```diff
@@ -81,6 +77,16 @@
     title: "How to add a model to 🤗 Transformers?"
   - local: add_new_pipeline
     title: "How to add a pipeline to 🤗 Transformers?"
+  - local: fast_tokenizers
+    title: "Using tokenizers from 🤗 Tokenizers"
```

Comment:
This change does not seem to be linked to this PR.

Reply:
Good catch!must have happened during rebase - I'll remove it.